### PR TITLE
UpdateWorklog method

### PR DESCRIPTION
**Pull request by prugala**
# PR Description

Add UpdateWorklog method:
https://docs.atlassian.com/software/jira/docs/api/REST/7.1.2/#api/2/issue-updateWorklog

# Checklist

* [x] Tests added
  * [x] Good Path
  * [ ] Error Path
* [x] Commits follow conventions described here:
  * [x] [https://conventionalcommits.org/en/v1.0.0-beta.4/#summary](https://conventionalcommits.org/en/v1.0.0-beta.4/#summary)
  * [x] [https://chris.beams.io/posts/git-commit/#seven-rules](https://chris.beams.io/posts/git-commit/#seven-rules)
* [x] Commits are squashed such that
  * [x] There is 1 commit per isolated change
* [x] I've not made extraneous commits/changes that are unrelated to my change.

### CODE REVIEW

**Comment by prugala** [comment]
@andygrunwald could you look at the changes and give me a comments or merge it? :)

File: issue.go [hunk 1]
```diff
@@ -867,6 +867,33 @@ func (s *IssueService) AddWorklogRecord(issueID string, record *WorklogRecord, o
 	return responseRecord, resp, nil
 }
 
+// AddWorklogRecord adds a new worklog record to issueID.
```
Comment (ghostsquad) [inline issue.go]: This comment doesn't match the method name

Comment (prugala) [reply issue.go]: Oh, I changed it.

**Comment by prugala** [comment]
done

**Pull request merged**